README.md

```diff
@@ -10,14 +10,15 @@
 </div>
 
 ## So what is Yatopia?
-Yatopia combines the best patches from many [Paper](https://github.com/PaperMC/Paper) forks and optimization mods, as well as many unique optimizations. We borrow some of our patches from the following repos:
+Yatopia combines the best patches from many [Paper](https://github.com/PaperMC/Paper) forks and optimization mods, as well as many unique optimizations. We use optimzations from the following repos:
```

Comment:
I like the change in wording here

Reply:
It's @budgidiere that edited the `README.md` and I also like the new wording too